django/contrib/contenttypes/management.py

```diff
@@ -82,3 +85,9 @@ def update_contenttypes(app_config, verbosity=2, interactive=True, using=DEFAULT
         else:
             if verbosity >= 2:
                 print("Stale content types remain.")
+
+
+def remove_contenttypes(app):
```

Comment:
This change was requested and it's submitted as requested. If you guys are having differences over this I suggest we just be open to the idea that any changes you want can always be made later. I don't think changing this again will settle the issue and frankly I'd like to get the change in sooner than later.